tests/testthat/test_edit_attribute.R

```diff
@@ -0,0 +1,43 @@
+testthat::context("edit_attribute")
```

Comment:
You don't actually need `testthat::` here, but it's ok to leave it. The way testing works is that `devtools::check` will look inside the `tests` folder and start by running `testthat.R`, which loads the `testthat` and `datamgmt` packages. Then, it goes into the `testthat` folder and continues testing all the files there.